src/components/v2/Layout/Header/styles.ts

```diff
@@ -3,11 +3,48 @@ import { useTheme } from '@mui/material';
 
 export const useStyles = () => {
   const theme = useTheme();
-  return css`
-    background-color: ${theme.palette.background.default};
-    width: ${theme.shape.layoutOffset.width};
-    margin-left: ${theme.shape.layoutOffset.ml};
-    background-image: none;
-    box-shadow: none;
-  `;
+  return {
+    appBar: css`
+      background-image: none;
+      background-color: transparent;
+      box-shadow: none;
+    `,
+    toolbar: css`
+      justify-content: space-between;
+      display: flex;
+
+      /* @TODO Remove when the ConnectButton is refactored */
+      .connect-btn {
+        height: auto;
+        margin: ${theme.spacing(2)};
+        padding: ${theme.spacing(1.5)} ${theme.spacing(3)};
+        width: 10.3rem;
+      }
+      ${theme.breakpoints.down('md')} {
+        display: none;
+      }
+    `,
+    rightItemPaper: css`
+      margin: ${theme.spacing(2)};
+      justify-content: center;
+      align-items: center;
+      padding: ${theme.spacing(1)};
+      display: inline-flex;
+      width: 10.3rem;
+    `,
+    coinInfo: css`
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      width: 10.3rem;
+      margin: ${theme.spacing(2)};
+      padding: ${theme.spacing(1.5)} ${theme.spacing(3)};
+      > * {
+        margin: ${theme.spacing(0.5)};
+      }
+      p {
+        color: ${theme.palette.text.primary};
+      }
+    `,
```

Comment:
Some of the margins are incorrect (space between the button left border and the coin icon too big, space between the coin icon and the amount too small) and there's a box shadow being applied to the buttons that isn't present in the designs:
<img width="216" alt="Screenshot 2022-03-23 at 09 32 30" src="https://user-images.githubusercontent.com/44675210/159668302-a4d6633f-ccf2-4fa9-8de6-6d82d822644b.png">

<img width="627" alt="Screenshot 2022-03-23 at 09 31 34" src="https://user-images.githubusercontent.com/44675210/159668311-50860889-94f7-4282-af17-dcffc2aa5769.png">